docs/packages/pkg/CppJwt.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    CppJwt
+
+.. index::
+  single: JWT ; CppJwt
+
+.. _pkg.CppJwt:
+
+CppJwt
+======
+
+-  `Official <https://github.com/QuoineFinancial/cpp-jwt>`__
```

Comment:
That's not an official repository. Upstream is https://github.com/arun11299/cpp-jwt

Reply:
There are many versions of CppJwt. All of them carry unwelcome intrinsic dependencies, such as copy/pasted versions of nlohmann_json or rapidjson. It's actually difficult to know which is the "original". Therefore, I am stating that from now on, *this* is the original. Because it is the only version which allows package dependency injection via cmake/hunter.

Reply:
> It's actually difficult to know which is the "original"

I think in this case it's pretty simple:
- https://github.com/arun11299/cpp-jwt/blob/v1.1.1/include/jwt/jwt.hpp#L2
```
Copyright (c) 2017 Arun Muralidharan
```
- https://github.com/arun11299
```
Arun Muralidharan
arun11299
```

This version is official:
- https://github.com/arun11299/cpp-jwt

Reply:
I've created a fork with branch `hunter-1.1.1`:
- https://github.com/hunter-packages/cpp-jwt/tree/hunter-1.1.1

Please send patches there.